samples/sensor/kx022/src/main.c

```diff
@@ -0,0 +1,311 @@
+/*
+ * Copyright (c) 2021 G-Technologies Sdn. Bhd.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/sensor.h>
+
+#define MAX_TEST_TIME 1500
+#define SLEEPTIME 300
+#define KX022_TILT_POS_FU 0x01
```

Comment:
not good to declare these thing in application, if these are necessary they should be in a header